packages/ui-components/src/styles/tokens.scss

```diff
@@ -122,16 +132,75 @@ $crl-base-divider: $crl-neutral-5;
   }
 }
 
-/**
- Variables
-*/
-// This variable reflects the base gutter value
-// specified by our design system.
-$crl-gutters: 8px;
+@mixin crl-transition($props_...) {
+  /**
+    Desc:
+      Helper for maintaining consistent transitions.
+
+    Example usage:
+      ```
+      .my-css-class {
+        @include crl-transition(color, border-color, height);
+      }
+      ```
+  */
+  transition-property: $props_;
+  transition-duration: $crl-transition-duration;
+  transition-timing-function: ease-in-out;
+}
+
+@mixin crl-box-shadow($offset_h_, $offset_v_, $blur_, $spread_, $color_) {
+  /**
+    Desc:
+      Helper for cross-browser box-shadows.
+
+    Example usage:
+      ```
+      .my-css-class {
+        @include crl-box-shadows(0, 2px, 4px, 0, $crl-blue-5);
+      }
+      ```
+  */
+  -webkit-box-shadow: $offset_h_ $offset_v_ $blur_ $spread_ $color_;
+  -moz-box-shadow: $offset_h_ $offset_v_ $blur_ $spread_ $color_;
+  box-shadow: $offset_h_ $offset_v_ $blur_ $spread_ $color_;
+}
 
 /**
  Functions
 */
+@function crl-lighten($color) {
+  /**
+    Desc:
+      Helper for consistent lightening of default tone,
+      for example in lightening a button in a disabled state.
+
+    Example usage:
+      ```
+      .my-css-class {
+        background-color: crl-lighten($crl-base-green);
+      }
+      ```
+  */
+  @return mix(white, $color, 70%);
```

Comment:
Update to 90%.

Tried 70% in managed console and it's way too saturated still.  90% sits properly against the primary tone.